phpBB/assets/javascript/supply_username.js

```diff
@@ -19,5 +19,4 @@ $(document).ready(function(){
 
 		}
 	});
-
```

Comment:
I am pretty sure @CHItA didn't mean that you should remove this line, but add a new one after });